apps/web/src/pages/templates/components/SmsBubble.tsx

```diff
@@ -0,0 +1,159 @@
+import styled from '@emotion/styled';
+import { Skeleton } from '@mantine/core';
+import { colors, PencilOutlined } from '@novu/design-system';
+import { MouseEventHandler, useState } from 'react';
+
+const SmsBubbleHolder = styled.div`
+  position: relative;
+`;
+
+const SmsBubbleContainer = styled.div<{ isBlur: boolean; isError: boolean }>`
+  position: relative;
+  width: fit-content;
+  min-width: 160px;
+  min-height: 36px;
+  padding: 8px 12px;
+  border-radius: 20px;
+  background: ${({ isError }) => (isError ? colors.error : '#51ba52')};
+  color: ${colors.white};
+  font-size: 14px;
+  line-height: 20px;
+  filter: ${({ isBlur }) => (isBlur ? 'blur(4px)' : 'none')};
+
+  &::before {
+    content: '';
+    position: absolute;
+    bottom: -2px;
+    right: -7px;
+    height: 20px;
+    border-right: 20px solid ${({ isError }) => (isError ? colors.error : '#51ba52')};
+    border-bottom-left-radius: 16px 14px;
+    transform: translate(0, -2px);
+  }
+
+  &::after {
+    content: '';
+    position: absolute;
+    z-index: 1;
+    bottom: -2px;
+    right: -46px;
+    width: 16px;
+    height: 20px;
+    background: ${({ theme }) => (theme.colorScheme === 'dark' ? '#4b4b51' : colors.white)};
+    border-bottom-left-radius: 10px;
+    transform: translate(-30px, -2px);
+  }
```

Comment:
the CSS magic to render the message bubble tail

Reply:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**🎯 suggestion:** Could you please add some comments to the code to explain the purpose / use case of this magic?‏